tests/unit_tests/init.js

```diff
@@ -1014,5 +1020,73 @@ describe('init tests', () => {
 
       fs.unlinkSync('test_config.conf.js');
     });
+
+    test('generateConfig with js with allowAnonymousMetrics', () => {
+      mockery.registerMock(
+        './logger',
+        class {
+          static error() {}
+        }
+      );
+
+      mockery.registerMock(
+        'uuid',
+
+        class {
+          static v4() {
+            return '3141-5926-5358-9793';
+          }
+        }
+      );
+
+      const answers = {
+        language: 'js',
+        backend: 'local',
+        browsers: ['chrome', 'firefox'],
+        defaultBrowser: 'firefox',
+        allowAnonymousMetrics: false
```

Comment:
Since `allowAnonymousMetrics` is set to `false` here, test title should contain *without* instead of *with*. Same with the next test.

Reply:
updated.